opds_import.py

```diff
@@ -273,7 +273,6 @@ def import_from_metadata(
                 even_if_no_author=even_if_no_author,
             )
             if work:
-                work.calculate_presentation()
```

Comment:
This is no longer necessary since LicensePool.calculate_work always calls work.calculate_presentation. Before, there were cases when we would call calculate_presentation twice.